pkg/connector/handlematrix.go

```diff
@@ -98,6 +98,24 @@ func (tc *TwitterClient) HandleMatrixMessage(ctx context.Context, msg *bridgev2.
 			uploadMediaParams.MediaCategory = "dm_gif"
 		}
 
+		if content.MsgType == event.MsgAudio {
+			user := tc.getCachedUserInfo(ParseUserLoginID(tc.userLogin.ID))
+			var userAvatar []byte
+			if user != nil {
+				userAvatar, err = user.Avatar.Get(ctx)
+				if err != nil {
+					return nil, err
+				}
+			}
```

Comment:
Is it possible to use a hardcoded image instead of the avatar? (like does it look sensible on native twitter)